src/main/java/com/microsoft/azure/spring/data/documentdb/repository/support/DocumentDbRepositoryFactoryBean.java

```diff
@@ -28,13 +29,11 @@
     private boolean mappingContextConfigured = false;
 
 
-    public DocumentDbRepositoryFactoryBean(Class<? extends T> repositoryInterface,
-                                           DocumentDbOperations operations) {
-
+    public DocumentDbRepositoryFactoryBean(Class<? extends T> repositoryInterface) {
         super(repositoryInterface);
-        this.operations = operations;
     }
 
+    @Autowired
```

Comment:
just to share, i remembered that i ever met issue that this set method is not called at initialize, maybe in docdb test.  but mongo works..

Reply:
Just FYI. The implementation Neo4jDB add this Autowired annotation, as it implements different interface... we may take a look into it in future.